docs/carbon-aware-webapi.md

```diff
@@ -316,6 +317,49 @@ The response is an array of CarbonIntensityDTO objects which each have a locatio
 ]
 ```
 
+### GET emissions/locations
+
+This endpoint lists all the supported locations that the datasources potentially can have access to. This information is coming from the `location-source/json` files, that contain dictionaries in the form of <A Location key name, GeoCoordinates>, for instance:
+
+```json
+{
+  "eastus": {
+    "Latitude": 37.3719,
+    "Longitude": -79.8164,
+    "Name": "eastus"
+  },
+  ...
+  "switzerlandnorth":{
+    "Latitude": 47.451542,
+    "Longitude": 8.564572,
+    "Name": "switzerlandnorth"
+  }
+}
+```
+
+`Emissions` and `Forecasts` endpoints would use the location's `key` name returned by the `emssions/locations` endpoint.
+
+**Note**: If there is ***a location key name*** collision while loading two or more location json files and there is no `LocationDataSourcesConfiguration` `Prefix` and/or `Delimiter` settings, the system would change the collision key name by appending an underscore character and a sequence number to it (i.e. `<key name>`***_seqnumber***). For instance two location json files having `swedencentral` as key, after loading, it would become:
```

Comment:
Question about this, if there is a collision would the user be told? Or is the idea that if they were to query the `locations` endpoint, then they would see the duplicated keys with non-colliding names and know from there